styles/faas.css

```diff
@@ -0,0 +1,3 @@
+.faas-form input:not([type=checkbox]):not([type=radio]):not([type=submit]), .faas-form select, .faas-form textarea {
+  font-size: 16px;
+}
```

Comment:
I don't think we should put any styles in until we're ready

Reply:
Without any styles there are linter error, but I could bypass those if you think it is best.